src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,15 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  return String(number).split("").reverse().map(Number);
+
```

Comment:
You dont need to include blank lines at the end of a function, this should be taken out